app/services/organization_service.rb

```diff
@@ -0,0 +1,12 @@
+class OrganizationService
+  def save_organization_and_cities_organization(organization: Organization.new,
```

Comment:
Almost any service should have `def initialize` where you prepare all instance variables you pass on to a service. If not - this is a utility library (some calculations, etc).